mne/utils/docs.py

```diff
@@ -1995,6 +1995,23 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     False.
 """
 
+docdict['notes_2d_backend'] = """\
+MNE-Python provides two different backends for signal-browsing plots (i.e.,
+:meth:`raw.plot()<mne.io.Raw.plot>`, :meth:`epochs.plot()<mne.Epochs.plot>`,
+and :meth:`ica.plot_sources()<mne.preprocessing.ICA.plot_sources>`). One is
+based on :mod:`matplotlib` and the other is based on
+:doc:`PyQtGraph<pyqtgraph:index>`. You can set the backend for the
+duration of a Python session using :func:`mne.viz.set_browser_backend`, or you
+can change the backend temporarily with the context manager
+:func:`mne.viz.use_browser_backend`.
```

Comment:
I think it would be nice if this note mentioned how to permanently set the backend.